app/server.go

```diff
@@ -213,10 +213,10 @@ func NewServer(options ...Option) (*Server, error) {
 		}
 	})
 
-	mlog.Info(fmt.Sprintf("Current version is %v (%v/%v/%v/%v)", model.CurrentVersion, model.BuildNumber, model.BuildDate, model.BuildHash, model.BuildHashEnterprise))
-	mlog.Info(fmt.Sprintf("Enterprise Enabled: %v", model.BuildEnterpriseReady))
+	mlog.Info("Current ", mlog.String("version", model.CurrentVersion), mlog.String("buildNumber", model.BuildNumber), mlog.String("buildDate", model.BuildDate), mlog.String("buildHash", model.BuildHash), mlog.String("enterprise", model.BuildHashEnterprise))
```

Comment:
I don't think this change is needed, it doesn't improve logging

Reply:
I know logging semantics can be fairly subjective, but this change does seem to fit the intent of the going towards structured logging. Even though it's wordier, being able to pick out individual keys and values is useful.

Reply:
@reflog @gabrieljackson Thanks for the feedback, I  would work on it.